setup.py

```diff
@@ -44,6 +44,10 @@
             "matplotlib",
             "openml==0.10.2",
         ],
+        "spark": [
+            "pyspark>=3.0.0",
+            "joblibspark",
```

Comment:
We should add some version constraints here so we don't get caught out on major upgrades